OrbitGrpcProtos/capture.proto

```diff
@@ -191,5 +235,6 @@ message CaptureEvent {
     InternedTracepointInfo interned_tracepoint_info = 9;
     TracepointEvent tracepoint_event = 10;
     IntrospectionScope introspection_scope = 12;
+    GpuQueueSubmission gpu_queue_submission = 13;
```

Comment:
I would suggest moving it right after `GpuJob`, it makes sense to me if they are next to each other.

Reply:
Done.

Reply:
I meant also here in `message CaptureEvent` (even more now that the order of definition of the messages is like that).